src/components/ModalCheckout/modalCheckout.css

```diff
@@ -0,0 +1,36 @@
+.checkoutForm {
+  padding: 20px 30px;
+}
+
+.modalTitle {
+  text-align: center;
+  font-size: 24px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.modalSubmit {
+  background-color: lightgray;
+  border: solid 2px rgb(97, 96, 96);
+  border-radius: 20px;
+  outline: none;
+  width: 40%;
+  margin: 20px auto 0px auto;
+  display: flex;
+  justify-content: center;
```

Comment:
Please, set CSS properties in a proper order, e.g.:
[CSS-order-properties](https://9elements.com/css-rule-order/#:~:text=Order%20of%20CSS%20layouting%20as,%3A%20position%20%2C%20float%20%2C%20display%20.)